docs/model_quality.mdx

```diff
@@ -0,0 +1,104 @@
+To effectively understand how accurate an emissions "measurement" is, given the many layers of assumptions and allocations involved in the calculation, is critical both for assessing provider quality and as a gating factor to incorporate data into corporate and regulatory reporting.
+
+## Model Quality Components
+
+Each modeled request should include:
+- Grid mix model quality (1-5)
+- Organization model quality (1-5) 
+- Property model quality (1-5)
+- Ad stack / ad platform quality (1-5)
+- Input granularity score (1-5)
+
+## Grid mix
+
+Grid intensity can fluctuate significantly on an hourly basis due to the variable nature of both renewable energy sources (sun, wind) and electricity demand. To achieve effective decarbonization, having hourly data is critical. However, this isn't broadly available in many countries.
+
+| Timescale | Geographical fit | Model Quality |
```

Comment:
how do you think we should represent the model quality for countries where we do not get any country specific grid mix information and we need to fallback to a WW model? i'd think that should be lower than annual + country

Reply:
updated to reflect this feedback